q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+#include <string>
+#include <vector>
+
+
+
+/* ****************************************************************************
+*
+* wsStrip - 
+*/
+static char* trim(char* s)
+{
+  // 1. beginning
+  while ((*s == ' ') || (*s == '\t') || (*s == '\n'))
+  {
+    ++s;
+  }
+
+  if (*s == 0)
+  {
+    return s;
+  }
+
+  // 2. end
+  char* end = &s[strlen(s) - 1];
+  while ((end > s) && ((*end == ' ') || (*end == '\t') || (*end == '\n')))
+  {
+    *end = 0;
+    --end;
+  }
+
+  return s;
+}
+
+
+
+/* ****************************************************************************
+*
+* qStringToBson - 
+*/
+int qStringToBson(char* in)
```

Comment:
This function to be moved to mongoBackend somewhere (preferribly to its own file) and this file should be removed.